src/main/kotlin/pl/edu/agh/gem/external/dto/InternalGroupResponse.kt

```diff
@@ -3,13 +3,23 @@ package pl.edu.agh.gem.external.dto
 import pl.edu.agh.gem.internal.model.Group
 
 data class InternalGroupResponse(
```

Comment:
will this change affect other services that use old version? (should we change all clients?)

Reply:
Since we've only added new fields, there won’t be any issues with backward compatibility. Jackson will not serialize these fields